### DISCUSSION TITLE
The latest diffusers version? 0.29.0? 0.30.0?

### DISCUSSION THREAD

**Discussion by robertchen245** [huggingface/diffusers#8582] [Q&A]
The latest version I could get from pip is 0.29.0. Where is 0.30.0?

**Comment by tolgacangoz** [comment]
[The latest stable release of `diffusers` is 0.29.0](https://github.com/huggingface/diffusers/tags). 0.30.0 is in the development phase now. [You can install *current* 0.30.0.dev0 by installing it from the source.](https://huggingface.co/docs/diffusers/main/en/installation#install-from-source)

**Reply by robertchen245** [reply]
much appreciate
